src/qt/intro.cpp

```diff
@@ -302,7 +302,10 @@ void Intro::startThread()
 
     connect(executor, &FreespaceChecker::reply, this, &Intro::setStatus);
     connect(this, &Intro::requestCheck, executor, &FreespaceChecker::check);
-    /*  make sure executor object is deleted in its own thread */
+    connect(ui->prune, &QCheckBox::toggled, [this](bool prune_checked) {
+        UpdatePruneLabels(prune_checked);
+        Q_EMIT requestCheck();
+    });
```

Comment:
In commit "gui: Fix intro dialog when prune is toggled" (f07dd7c69ed7a5245f79662100c859286557b73b)

Why trigger a new disk space check here? Can you add explanatory comments? I think I understand updating the labels because the label text depends on the check state, but I don't get the disk space thing.

It would also be good to update commit description to say what behavior was before and after this change, and how it works.

Reply:
> Why trigger a new disk space check here?

Reworked. No unneeded disk space check anymore.